web/modules/custom/usagov_benefit_category_search/js/benefitSearch.js

```diff
@@ -241,6 +241,8 @@ function BenefitSearch(benefitsPath, lifeEventsPath, assetBase, docLang, labels,
     myself.showResults();
     // update browser history so that bookmarks work
     myself.updateHistory();
+    // send data to GTM for a success in applying selections
+    dataLayer.push({'event': 'category_apply_success'});
```

Comment:
I tested locally and see the dataLayer object is available via global scope, so this should work. However, if the dataLayer object isn't available for some reason, it'll likely break our search page. To be safe, let's wrap it in an if

```suggestion
    if (dataLayer != null) {
        dataLayer.push({'event': 'category_apply_success'});
    }
```

Reply:
good catch.. updated with the if statement